lib/rubocop/cop/lint/multiple_comparison.rb

```diff
@@ -7,25 +7,26 @@ module Lint
       # multiple value. However, we can't use the comparison in Ruby. However,
       # the comparison is not syntax error. This cop checks the bad usage of
       # comparison operators.
+      # It allows multiple comparison using `&`, `|`, and `^` set operation operators.
+      #
+      # The cop is marked as unsafe auto-correction because comparison of `1 < 2 < 3` is
```

Comment:
I think this example is confusing - I get it, but I don't get how is it relevant to the cop and its safety. It'd be better to illustrate this with some case where it's more obvious that someone intentially wanted a chain a comparsion ops.